src/30_Advanced/Video_Carousels_with_amp-carousel.html

```diff
@@ -0,0 +1,115 @@
+<!--
+  #### Introduction
+
+  While it's possible to [integrate videos](https://ampbyexample.com/advanced/integrating_videos_in_amp_an_overview/) in AMP HTML files, when having different videos that you want to display to users, these could be embedded inside `amp-carousel`. This example demonstrates how to create a responsive video carousel, which unifies video integrations using different aspect ratios.
```

Comment:
Not sure I understand the first sentence. Maybe: "The `amp-carousel` component works not only great for image galleries, but also for video galleries. The challenge when implementing a video gallery is to make sure that all videos fit into the carousel. This sample shows how to implement a massive video carousel using all available AMP video integrations."